scipy/spatial/qhull.pyi

```diff
@@ -5,7 +5,7 @@ Static type checking stub file for scipy/spatial/qhull.pyx
 from typing import List, Tuple, Any, Dict
 
 import numpy as np
-from numpy.typing import ArrayLike
+from numpy.typing import ArrayLike, NDArray
```

Comment:
[`npt.NDArray`](https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray) is more compact alias for `np.ndarray[Any, np.dtype[~ScalarType]]`.